packages/nimble-components/src/anchor-tabs/index.ts

```diff
@@ -0,0 +1,286 @@
+import { attr, observable } from '@microsoft/fast-element';
```

Comment:
For files that used FAST as a starting point, I think it would be useful to add a comment that links back to the original source. Among other benefits, it might help remind us to refer back to FAST's implementation if we find ourselves making changes to ours.

Reply:
Related: any guidance you can give reviewers about which parts of this file have changed? I didn't closely review all the logic about handling events and setting active tab under the assumption that it was FAST code. But if you had to do anything significant in this file other than deleting unneeded code it would be helpful to highlight it.

Reply:
Added a comment at the top of the file. Really there's no practical way to diff the files because they have changed so much, but I've tried to enumerate all the things that changed.